packages/back-end/src/scim/users/createUser.ts

```diff
@@ -5,31 +5,47 @@ import {
   convertMemberToManagedByIdp,
   expandOrgMembers,
 } from "../../services/organizations";
-import { OrganizationInterface } from "../../../types/organization";
-import {
-  ScimError,
-  ScimUser,
-  ScimUserPutOrPostRequest,
-} from "../../../types/scim";
+import { OrganizationInterface, MemberRole } from "../../../types/organization";
+import { ScimError, ScimUser, ScimUserPostRequest } from "../../../types/scim";
 import {
   createUser as createNewUser,
   getUserByEmail,
 } from "../../services/users";
 
+export function isRoleValid(role: MemberRole) {
```

Comment:
Anyone have a suggestion on a better pattern here that doesn't require us to manually create this array? As we add roles, this can get out of date.